x11-terms/terminator/terminator-1.92.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_6 python3_7 )
+inherit eutils distutils-r1 virtualx xdg
+
+DESCRIPTION="Multiple GNOME terminals in one window"
+HOMEPAGE="https://github.com/gnome-terminator/terminator"
+SRC_URI="https://github.com/gnome-terminator/terminator/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="dbus"
+
+RDEPEND="
+	>=x11-libs/gtk+-3.16:3
+	>=dev-libs/glib-2.32:2
+	dev-libs/keybinder:3[introspection]
+	dev-python/psutil
+	x11-libs/vte:2.91[introspection]
+	dbus? ( sys-apps/dbus )
+	dev-python/configobj[${PYTHON_USEDEP}]
+	dev-python/pycairo[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+"
+DEPEND="
```

Comment:
it doesn't need dev-libs/gtk+ etc. to build?

Reply:
During build it just copies files around and converts .py to .pyc

However, during test phase it might need gtk+ and the rest when executing the test... Should I add IUSE=test, and put it to either RDEPEND or BDEPEND depending on `$(use test)`?